lib/features/game/views/widgets.dart

```diff
@@ -78,22 +82,28 @@ class HeaderText extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.fromLTRB(16, 30, 16, 0),
-      child: Text(text, style: TextStyle(fontSize: 25, color: KColors.basedBlackColor)),
+    return FittedBox(
+      fit: BoxFit.fitWidth,
+      child: Padding(
+        padding: const EdgeInsets.fromLTRB(16, 30, 16, 0),
+        child: Text(text, style: TextStyle(fontSize: 35, color: KColors.basedBlackColor)),
+      ),
     );
   }
 }
 
 class ResultListItem extends StatelessWidget {
   final PlayerInfo playerInfo;
+  final int index;
 
-  const ResultListItem({required this.playerInfo});
+  const ResultListItem({required this.playerInfo, required this.index});
 
   static const _resultTextStyle = TextStyle(color: Colors.white, fontSize: 18, fontWeight: FontWeight.bold);
 
   @override
   Widget build(BuildContext context) {
+    final int formattedScore = playerInfo.score.toInt() * 100;
```

Comment:
Shouldn't that math be reversed? Like `final int formattedScore = (playerInfo.score * 100).toInt();`

Reply:
done